codec-http2/src/main/java/io/netty/handler/codec/http2/Http2CodecUtil.java

```diff
@@ -202,26 +202,7 @@ public static ByteBuf toByteBuf(ChannelHandlerContext ctx, Throwable cause) {
      * Reads a big-endian (31-bit) integer from the buffer.
      */
     public static int readUnsignedInt(ByteBuf buf) {
-        return (buf.readByte() & 0x7F) << 24 | (buf.readByte() & 0xFF) << 16
-                | (buf.readByte() & 0xFF) << 8 | buf.readByte() & 0xFF;
-    }
-
-    /**
-     * Writes a big-endian (32-bit) unsigned integer to the buffer.
-     */
-    public static void writeUnsignedInt(long value, ByteBuf out) {
-        out.writeByte((int) (value >> 24 & 0xFF));
-        out.writeByte((int) (value >> 16 & 0xFF));
-        out.writeByte((int) (value >> 8 & 0xFF));
-        out.writeByte((int) (value & 0xFF));
-    }
-
-    /**
-     * Writes a big-endian (16-bit) unsigned integer to the buffer.
-     */
-    public static void writeUnsignedShort(int value, ByteBuf out) {
-        out.writeByte(value >> 8 & 0xFF);
-        out.writeByte(value & 0xFF);
+        return buf.readInt() & 0x7fffffff;
```

Comment:
Please make this `7FFFFFFF` as e use uppercase here as well:
https://github.com/Scottmitch/netty/blob/9bc6c216e67cc5b5876197e90ddad78039889a88/codec-http2/src/main/java/io/netty/handler/codec/http2/Http2CodecUtil.java#L96

Or make the other lowercase.